docs/INTEGRATION.md

```
@@ -0,0 +1,790 @@
+- **Error Mapping:** `distros/kubernetes/nvsentinel/charts/gpu-health-monitor/files/dcgmerrorsmapping.csv`
+- **Quarantine Rules:** `distros/kubernetes/nvsentinel/charts/fault-quarantine/values.yaml`
+- **Module Config:** `distros/kubernetes/nvsentinel/values.yaml`
+
+### Code Locations
+
+- **Condition Setting:** `platform-connectors/pkg/connectors/kubernetes/process_node_events.go`
+- **Taint Application:** `fault-quarantine/pkg/informer/k8s_client.go`
+- **Drain Logic:** `node-drainer/pkg/drainer/drainer.go`
+- **Remediation Triggering:** `fault-remediation/pkg/remediation/remediation.go`
+
+## Related Documentation
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)